src/index.js

```diff
@@ -6,9 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.scss';
+import reportWebVitals from './reportWebVitals';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Planthor-Team/Planthor_ClientFrontEndWebApp/pullRequest?prid=7095961)
```suggestion
import App from "./App.js";
```